guacamole-docker/bin/start.sh

```diff
@@ -536,6 +536,21 @@ END
 
 }
 
+##
+## Adds properties to guacamole.properties which configure the TOTP two-factor
+## authentication service.
```

Comment:
Unlike Duo, TOTP isn't an authentication service. There may be a more correct way to document this.

Reply:
Done, thank you for your review 👍